src/ChannelServer/World/WorldManager.cs

```diff
@@ -177,5 +177,23 @@ public Character GetCharacterByTeamName(string teamName)
 
 			return null;
 		}
+
+		/// <summary>
+		/// Returns all Characters that are currently online.
+		/// </summary>
+		public Character[] GetCharacters()
```

Comment:
Nothing wrong with this code, but there's a shorter and potentially slightly faster way to do it if you're interested:
```csharp
lock (_mapsLock)
	return _mapsId.Values.SelectMany(a => a.GetCharacters()).ToArray();
```